tools/release.rb

```diff
@@ -100,13 +125,29 @@ def self.generate_release_version(release_type)
     "#{major}.#{minor}.#{patch}"
   end
 
+  def self.generate_development_version
+    # The development version RPMs have the following version number scheme:
+    # <base version>.<timestamp><os>git<short git hash>
+    timestamp = Time.now.strftime("%Y%m%dT%H%M%SZ")
+    commit_id = Cheetah.run("git", "rev-parse", "--short", "HEAD", :stdout => :capture).chomp
+
+    "#{Release.version}.#{timestamp}#{build_dist.gsub(/[._]/, "")}git#{commit_id}"
+  end
+
   private
 
+  def prepare_and_publish
+    prepare
+
+    # Build gem and send everything to IBS
+    Rake::Task['osc:commit'].invoke
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.